### ISSUE TITLE
Bower js files to dist?

### ISSUE THREAD

**Issue by bichotll** [randylien/generator-react-gulp-browserify#39]
Hi,
I've just noticed that you are copying the js files to dist?
I think that's not necessary as it already is building a vendor js file of all the third party ones.

https://github.com/randylien/generator-react-gulp-browserify/blob/master/app/templates/_gulpfile.js#L149

**Comment by codeofsumit** [comment]
That's correct for JS files. CSS files however are not being bundles into a vendor.css

This generator is pretty broken right now :-(
